var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -230,7 +232,7 @@ def configure_args(self):
         else:
             extra_args.append('--without-szlib')
 
-        if self.spec.satisfies('@1.10:'):
+        if self.spec.satisfies('@1.10:1.12.0'):
```

Comment:
Ah, I didn't realize we were already adding this, you can undo this change. Sorry for the confusion.

Reply:
No problem. I'll remove the change.